### PR TITLE
Added full build instructions for ubuntu

### PR DESCRIPTION
**Pull request by linuxraspilxc**
My little present for christmas.

Full instructions to build on Ubuntu.
I have always found it kinda irritating to get the dependencies right.

### CODE REVIEW

File: docs/BUILD.md [hunk 1]
```diff
@@ -28,6 +28,47 @@ For the build to pass you need to install the following from sources: [filteraud
...
+make
+sudo make install
+cd..
+
+git clone git://github.com/GrayHatter/uTox.git
+cd uTox/
+make
+make install
+
+ldconfig
```
Comment (cebe) [inline docs/BUILD.md]: why run ldconfig after everything is installed?

File: docs/BUILD.md [hunk 2]
```diff
@@ -28,6 +28,47 @@ For the build to pass you need to install the following from sources: [filteraud
...
+autoreconf -i
+./configure
+make
+sudo make install
+cd..
+
+git clone git://github.com/GrayHatter/uTox.git
+cd uTox/
+make
+make install
```
Comment (cebe) [inline docs/BUILD.md]: no sudo here?

**Comment by linuxraspilxc** [comment]
fixed it 
I needed to call ldconfig because if I don't do so, I get the following error:
"utox: error while loading shared libraries: libtoxav.so.0: cannot open shared object file: No such file or directory"

**Comment by GrayHatter** [comment]
@linuxraspilxc thanks mate! @subliun will love you!

**Comment by subliun** [comment]
@linuxraspilxc <3

**Comment by GrayHatter** [comment]
sorry, I though I merged this a long time ago!

**Pull request merged**